startpage_content.md

```diff
@@ -0,0 +1,62 @@
+This WIP PR is for preparing content for the start page.
+
+Please commit your ideas until 2019-05-11 so that we can discuss on the next web meeting about the design.
+
+Solutons could be:
+
+* screens (can take a lot of content)
+* carousel/slides (all at top, do not take much content)
+* carousel/slides with screens (teaser + _learn more_-button)
+* ...
+
+# Persistent content
+
+## Green Open Access
+_(Citation advantage of OA https://sparceurope.org/what-we-do/open-access/sparc-europe-open-access-resources/open-access-citation-advantage-service-oaca/ )_
+
+Many researchers do not use their right to make their papers freely available online, in addition to the paywalled version offered by traditional publishers.
+
+There are two major advantages of unpaywalling your papers:
+1. There are a lot of studies giving evidence, that freely available publications have a higher citation rate. 
+This is called the _Open Access Citation Advantage_. You can find a list of studies on https://sparceurope.org/what-we-do/open-access/sparc-europe-open-access-resources/open-access-citation-advantage-service-oaca/oaca-list/ 
+As a conclusion: Your research is not only more acknowledged, but pushes scientific advantage in general, because your research better spreads!
```

Comment:
I would first start with something general about articles being available to readers (peers / patients / the industry / the general public) and only mention citation rates in a second sentence (because citation rates are debatable metrics, researchers should not try to optimize them per se - they should rather try to increase the reach of their publications in a general sense).

Reply:
+1 Antonin for citations being only a secondary point. Several more comments:

- We need to have as persistent content a description of what the service proposes, before giving context about green OA. We currently have " Dissemin detects papers behind pay-walls and invites their authors to upload them in one click to an open repository." I think that's OK except we may want to add some term like "green OA" to this if we have more detailed explanations afterwards.
- I think the text should be shorter. I'd keep the current "Many researchers do not use their right to make their papers freely available online, in addition to the paywalled version offered by traditional publishers." and then follow by a list of advantages in a few words:
  - Ensure that libraries don't need to pay overpriced subscriptions to scientific journals.
  - Make sure that everyone can read your research, including people outside of established research labs
  - Give more impact to your research (just link to the relevant study)